src/styles/_catalog.scss

```
@@ -1,19 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  gap: 48px;
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 488px) and (max-width: 768px) {
```

```suggestion
  @media (min-width: 488px){
```

max-width can be removed as per my understanding. And also max-width should be 1px less than the  above min-width.

---

Use one condition at a time by putting media queries in the correct order.
Example:
default -> media (max-width: 1023px) -> media (max-width: 767px) -> media (max-width: 477px)